### PR TITLE
feat(sz): configurable navigation icons

### PR DESCRIPTION
**Pull request by luytena**

### CODE REVIEW

File: packages/distribution/addon/utils/inquiry-status.js [hunk 1]
```diff
@@ -41,16 +43,49 @@ function decorator(
...
+                  ...config,
+                  color: config.color[inquiryType] ?? config.color,
+                };
+              })
+              .filter(Boolean)[0]
+          : null;
+
+      if (inquiryAnswerStatus) {
+        return inquiryAnswerStatus;
+      }
```
Comment (anehx) [inline packages/distribution/addon/utils/inquiry-status.js:82]: Where's the config part for this?

**Review by anehx — CHANGES_REQUESTED** [review]
The tests for the inquiry status util absolutely needs to be extended. Otherwise nice work!

File: packages/distribution/addon/utils/inquiry-status.js (hunk 1 above)
Comment (luytena) [reply packages/distribution/addon/utils/inquiry-status.js:82]: Good point, I'll add one for the "confirm-inquiry-answer" work-item.

Comment (luytena) [reply packages/distribution/addon/utils/inquiry-status.js]: Added

**Comment by luytena** [comment]
> The tests for the inquiry status util absolutely needs to be extended. Otherwise nice work!

I extended the necessary tests

**Review by anehx — APPROVED** [review]
Awesome work! :tada:

**Pull request merged**